client/src/intl/en.json

```diff
@@ -0,0 +1,6 @@
+{
+  "71L0pp": {
```

Comment:
Is this ID random?

Reply:
Yeah - I documented this in the readme as well; it's producted by the `formatjs extract` command -- unfortunately this requires you to have to go back and put it back in later. I think there are alternatives to this through `babel` and/or `eslint` we could look into as well but this may have to do in the meantime.